examples/server-render/pages/index.js

```diff
@@ -0,0 +1,24 @@
+import Link from 'next/link'
+import fetcher from '../libs/fetcher'
+
+import useSWR from 'swr'
+
+export default function Home({ initialData }) {
+  const { data } = useSWR('/api/data', fetcher, { initialData })
+
+  return <div style={{ textAlign: 'center' }}>
+    <h1>Trending Projects</h1>
+    <div>
+    {
+      data ? data.map(project => 
+        <p key={project}><Link href='/[user]/[repo]' as={`/${project}`}><a>{project}</a></Link></p>
+      ) : 'loading...'
+    }
+    </div>
+  </div>
+}
+
+Home.getInitialProps = async () => {
+  const data = await fetcher('http://localhost:3000/api/data')
```

Comment:
Hmmm this won't work for deployments... should we use a public API as the example? e.g.: https://pokeapi.co

Reply:
You are right, I changed it to use Pokeapi and render a list of pokemon in `/` and a detail in `/[pokemon]`